src/styles/base/_base.scss

```diff
@@ -0,0 +1,27 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
```

Comment:
It's better to avoid setting `font-size` on the `html` element to a fixed pixel value. This can cause issues with accessibility and user preference settings. Consider using relative units like `em` or `rem`.